_shared/project/tox.ini

```diff
@@ -64,27 +64,36 @@ deps =
         {{- include("tox/deps", indent=4) -}}
     {% endif %}
 depends =
-{% if cookiecutter._directory == 'pyramid-app' %}
+{% if cookiecutter._directory in ['pyapp', 'pyramid-app'] %}
     coverage: tests
 {% else %}
     coverage: tests,py{{ "{" }}{{ python_versions()|rest|pyformat(PyFormats.MAJOR_MINOR_FMT)|separator(",") }}{{ "}" }}-tests
 {% endif %}
+{% if cookiecutter.get("db") == "yes" or include_exists("tox/allowlist_externals") %}
+allowlist_externals =
+{% if cookiecutter.get("db") == "yes" %}
+    tests,functests: sh
+{% endif %}
 {% if include_exists("tox/allowlist_externals") %}
-allowlist_externals = 
     {{- include("tox/allowlist_externals", indent=4) -}}
 {% endif %}
-{% if cookiecutter._directory == 'pyramid-app' or include_exists("tox/commands_pre") %}
+{% endif %}
+{% if cookiecutter._directory in ['pyapp', 'pyramid-app'] or include_exists("tox/commands_pre") %}
 commands_pre =
-    {% if cookiecutter._directory == 'pyramid-app' %}
+    {% if cookiecutter._directory in ['pyapp', 'pyramid-app'] %}
     pip-sync requirements/{env:TOX_ENV_NAME}.txt --pip-args '--disable-pip-version-check'
     {% endif %}
     {% if include_exists("tox/commands_pre") %}
         {{- include("tox/commands_pre", indent=4) -}}
     {% endif %}
 {% endif %}
 commands =
-{% if cookiecutter._directory == 'pyramid-app' %}
+{% if cookiecutter._directory in ['pyapp', 'pyramid-app'] %}
     dev: {posargs:supervisord -c conf/supervisord-dev.conf}
+{% if cookiecutter.get("db") == "yes" %}
+    tests: sh bin/create-db {{ cookiecutter.package_name }}_test
+    functests: sh bin/create-db {{ cookiecutter.package_name }}_functests
+{% endif %}
```

Comment:
This was another thing that was missing from the `pyramid-app` cookiecutter: it doesn't create the DBs if they don't exist yet.